File: klacks-web-api/klacks-web-api/Models/Calendar/HolydayRule.cs


using System;

namespace klacks_web_api.Models
{
  public class HolydayRule
  {
    public Guid Id { get; set; }
    public string Name { get; set; }
    public string Rule { get; set; }
    public string SubRule {  get; set; }
    public string Description { get; set; }
    public bool Visible { get; set; }
    public bool Paid { get; set; }

  }
}

File: src/app/core/holyday-rule.ts


export interface IHolydayRule {
  id: string;
  name: string;
  rule: string;
  subRule: string;
  description: string;
  visible: boolean;
  paid: boolean

}
export class HolydayRule implements IHolydayRule {
  id = '';
  name = '';
  rule = '';
  subRule = '';
  description = '';
  visible =false;
  paid =false;

}
